src/components/footer/footerNavbar.tsx

```diff
@@ -38,7 +38,7 @@ const FooterNavbar = () => {
                 size="4x"
                 className="text-white mt-3 mb-3"
               />
-              <p className="text-white text-decoration-none">How to download data<br />&nbsp;</p>
+              <p className="text-white text-decoration-none">How to Get Data Access<br />&nbsp;</p>
```

Comment:
Because we use Access and Submit in the top navbar, that's why we also updated the wording here.

Reply:
No I understand, I just mean whether 'How to access data' would be better wording (across every part of the site) than 'how to get data access', but regardless I will approve, since it is really minor